riotfile.py

```diff
@@ -112,6 +111,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "flake8-isort": latest,
                 "pygments": latest,
             },
+            venvs=[
+                Venv(
+                    name="flake8",
+                    command="flake8 {cmdargs} ddtrace/ tests/",
+                ),
+                Venv(
+                    name="flake8-hook",
```

Comment:
does this run when you do `riot run flake8`? since `run` takes a pattern not an explicit name